config/test.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
 module.exports = {
-  PORT: 3000
+  PORT: 3000,
+  DB_HOST: 'localhost',
+  DB_NAME: 'sfmovies_development',
```

Comment:
I don't remember if this is specified in Miyagi, but it might be worth creating a test DB separate from your development one; that way, things in your development db don't get wiped if you truncate tables in your tests (we do this for lob-api). Totally up to you though!

Reply:
Oh you're right bad Tim.